### PR TITLE
seed json data and csv data to postgres and mongo

### PR DESCRIPTION
**Pull request by andyoe0102**

### CODE REVIEW

File: db/seeds/04_game_join_companies.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+const seedFile = require('knex-seed-file');
+
 
 exports.seed = function(knex) {
   // Deletes ALL existing entries
   return knex('game_join_companies').del()
     .then(() => {
       // Inserts seed entries
-      return knex('game_join_companies').insert(generateGameJoinCompanies());
+      return knex.raw("COPY game_join_companies (id_game,id_developer,id_publisher,id_platform) FROM '/home/andyoe/Desktop/gameDescription/db/seeds/JoinTable_test.csv' WITH (FORMAT CSV, DELIMITER ',', HEADER)")
```
Comment (damienmjones) [inline db/seeds/04_game_join_companies.js:22]: Style thing, but...any reason why this one wasn't kept as a variable? It seems a little easier to read and abstract that way.

File: db/seeds/03_descriptions.js [hunk 2]
```diff
@@ -1,10 +1,96 @@
...
+
+
 
 exports.seed = function(knex) {
   // Deletes ALL existing entries
   return knex('descriptions').del()
     .then(() => {
       // Inserts seed entries
-      return knex('descriptions').insert(generateGameDesc());
+      return knex.raw("COPY descriptions (description,release_date) FROM '/home/andyoe/Desktop/gameDescription/db/seeds/descriptions_test.csv' WITH (FORMAT CSV, DELIMITER ',', HEADER);")
```
Comment (damienmjones) [inline db/seeds/03_descriptions.js:23]: Ah - I'm guessing that with your simplifications, it didn't seem necessary :)

File: db/seed_SDC_json.js [hunk 3]
```diff
@@ -66,54 +68,85 @@ let platformTable = [
...
 
-    cb();
+    return obj
+
+
+
+}
+
+
+function writeTenMillionJson(writer,encoding,callback){
```
Comment (damienmjones) [inline db/seed_SDC_json.js:143]: I feel like this function could use a bit of commenting.

**Review by damienmjones — APPROVED** [review]
Overall this looks good - I think in some places when code gets a but clever or reliant on external dependencies, commenting is definitely helpful for someone looking at the code for the first time :)

File: db/seeds/04_game_join_companies.js (hunk 1 above)
Comment (andyoe0102) [reply db/seeds/04_game_join_companies.js:22]: generateGameJoinCompnies is a function that returns an array of data. knex.raw reads the data directly from the csv file.

File: db/seed_SDC_json.js (hunk 3 above)
Comment (andyoe0102) [reply db/seed_SDC_json.js:143]: thanks for that suggestion, ill add comment on it!

**Pull request merged**